exercises/react/src/example.c

```diff
@@ -0,0 +1,210 @@
+#include <stdlib.h>
+#include "react.h"
+
+enum cell_kind {
+   kind_input,
+   kind_compute1,
+   kind_compute2,
+};
+
+struct child {
+   struct cell *cell;
+   struct child *next;
+};
+
+struct cb {
+   callback_id id;
+   callback f;
+   void *obj;
+   struct cb *next;
+};
+
+struct reactor {
+   struct child *first;
+   struct child *last;
```

Comment:
Oh you're right, I forgot that I need the ordering both in `destroy_reactor` and `cell_set_value`. Yeah, probably have to leave as-is then.